src/modules/admin/components/Tokens/index.ts

```diff
@@ -1,6 +1,6 @@
+export { default as TransferFundsDialog } from '../../../dashboard/components/TransferFundsDialog';
```

Comment:
Not sure why we exporting it here 🤔  @rdig What was the reason for those exports ealier?